### PR TITLE
known issue fixed for the certificate download as pdf and png

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

File: package.json
Comment (Vinod-V3) [inline package.json:1]: There is one more package.json file from where u have to remove this package

File: projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.ts [hunk 1]
```diff
@@ -107,31 +118,45 @@ export class CertificatePageComponent extends BackNavigationHandlerComponent {
...
-        URL.revokeObjectURL(url);
+        this.downloadPdf();
       } else {
+        this.downloadPng(canvas);
+      }
+    }
+  }
+
+  downloadPdf() {
+    fetch(this.projectDetails.certificate.pdfUrl)
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.ts:129]: Here without converting it into blob, can we use the same url from api response and achieve the same functionality?

Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.ts:129]: creating a temporary file to rename the file without using that name of the file is not changing.

File: package.json
Comment (farhanp1502) [reply package.json:1]: done

File: projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.ts (hunk 1 above)
Comment (Vinod-V3) [reply projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.ts:129]: ok

File: projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html [hunk 2]
```diff
@@ -12,7 +12,7 @@ <h4 class="text-lg " >{{projectDetails?.title}}</h4>
                 <div>
                     <button mat-flat-button class="!rounded-2xl primary-button" [matMenuTriggerFor]="menu"><mat-icon fontIcon="cloud_download"></mat-icon>{{"DOWNLOAD"| translate}}</button>
                     <mat-menu #menu="matMenu" >
-                        <button (click)="download('pdf')" mat-menu-item>{{"PDF"| translate}}</button>
+                        <button (click)="downloadPdf()" mat-menu-item>{{"PDF"| translate}}</button>
                         <button (click)="download('png')" mat-menu-item>{{"PNG"| translate}}</button>
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html]: do we need to pass the parameter?
Also change the function name to downloadSvg

Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html]: done

**Review by Vinod-V3 — APPROVED** [review]

**Pull request merged**